cli/internal/publish/pluginui.go

```diff
@@ -65,14 +81,36 @@ func UploadPluginUIAssets(ctx context.Context, c *cloudquery_api.ClientWithRespo
 	eg, egCtx := errgroup.WithContext(ctx)
 	eg.SetLimit(4)
 
+	var bundleAsset cloudquery_api.PluginUIAsset
 	for _, asset := range resp.JSON201.Assets {
+		if asset.Name == uiAssetBundleTarName {
+			bundleAsset = asset
+			continue
+		}
+
 		asset := asset
 		details := urlPathVsDetails[asset.Name]
 		eg.Go(func() error {
 			return hub.UploadFileWithContentType(egCtx, asset.UploadURL, details[0], details[1])
 		})
 	}
 
+	if bundleAsset.UploadURL == "" {
+		return errors.New("bundle asset URL not found in the response")
+	}
+
+	bundleReader, bundleWriter := io.Pipe()
+	defer bundleWriter.Close()
+
+	eg.Go(func() error {
+		defer bundleWriter.Close()
```

Comment:
This call is a duplicate, also in L103

Reply:
Fixed in https://github.com/cloudquery/cloudquery/pull/20116/commits/b1fcdbed4c55d556bf08926355161c232832490f

Reply:
The separate close cases were intentional. The one that was removed is needed for the reader to eventually hit EOF.

[c1a544e](https://github.com/cloudquery/cloudquery/pull/20116/commits/c1a544edd411fe64d963c0497829a127f9504ff4) makes this more explicit and adds better force cancelation for error cases.